packages/gafl-webapp-service/src/pages/payment/failed/route.js

```diff
@@ -17,7 +18,7 @@ const getData = async request => {
     codes: GOVUK_PAY_ERROR_STATUS_CODES,
```

Comment:
There does not appear to be any tests for the return value (possibly a snapshot? Might need to check with @philhunt-defra). Also there is an if statement which is not tested in this file - although I appreciate you did not create this file initially.